### ISSUE TITLE
Add periodically data update example for jvm-swing-javafx

### ISSUE THREAD

**Issue by fmvin** [alshan/lets-plot-mini-apps#2]
Please create an example for demonstrating plotting in realtime. For example math or rand function updates each second.
`jvm-swing-javafx` example is much preferable.

**Comment by alshan** [comment]
Ho @fmvin , this must be some trick with offscreen drawing of kind. Perhaps [CardLayout](https://docs.oracle.com/javase/tutorial/uiswing/layout/card.html) can help.
Lets-Plot itself doesn't support incremental updates at the moment so, every time the data is updated a brand new plot object must be created.

**Comment by fmvin** [comment]
@alshan, Thanks a  lot for clarification. There is no problem with creating new plot objects except an annoying chart blinking after each data update. Hope your proposal will be helpful.

**Comment by alshan** [comment]
BTW, just setting the color of the backing panel in your app to the same value as the color of the plot background can reduce blinking.

**Comment by fmvin** [comment]
Trick with CardLayout works exactly as I expected. On first data update I show first card and push a new chart to the second card. On next data update I show second card and push new chart to the first card. And so on.

**Comment by alshan** [comment]
Wow, glad to hear!
Do you think you can share your solution?

**Comment by fmvin** [comment]
I dont think this is a solution. It's just a naive trick with obvious limitations. Much better if native support of dynamic chart update will be added to Lets-Plot lib or to kotlin wrapper.
